media.go

```diff
@@ -35,11 +35,27 @@ func (mc *Client) UploadMedia(filePath, description, focus string) (*Attachment,
 	}
 	defer f.Close()
 
-	w := multipart.NewWriter(&b)
-	formWriter, err := w.CreateFormFile("file", filepath.Base(filePath))
+	return mc.UploadMediaReader(f, filepath.Base(f.Name()), description, focus)
+}
+
+// UploadMedia uploads data from the given reader and returns an attachment
```

Comment:
I think the name needs to be updated...
```suggestion
// UploadMediaReader uploads data from the given reader and returns an attachment
```